brambling/static/brambling/brambling.utils.js

```diff
@@ -0,0 +1,30 @@
+// Some simple Javascript utilities to be used wherever:
+
+;(function () {
+	"use strict";
+
+	// Filters to be used in Ractive mustaches:
+	brambling.filters = {
+		// Wraps a bit of text in an element:
+		highlight: function (needle, haystack, open_tag, close_tag) {
```

Comment:
I now have it returning undefined if the haystack isn't there, but having it return the original haystack if the needle isn't there.